tests/MongoDB.Driver.Core.Tests/Core/Configuration/ClusterSettingsTests.cs

```diff
@@ -56,6 +57,16 @@ public void constructor_should_throw_when_serverSelectionTimeout_is_negative()
             action.ShouldThrow<ArgumentException>().And.ParamName.Should().Be("serverSelectionTimeout");
         }
 
+        [Fact]
+        public void constructor_should_throw_when_localThreshold_is_negative()
+        {
+            var exception = Record.Exception(() => new ClusterSettings(localThreshold: TimeSpan.FromSeconds(-1)));
+
+            var argumentException = exception.Should().BeOfType<ArgumentOutOfRangeException>().Subject;
```

Comment:
Consider (it's how this construction usually looks like):
    
    var e = exception.Should ...
    e.ParamName ...

Reply:
Done.